src/pages/smart-contracts/clarity-values-guide.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Clarity Values Guide
+description: Learn how to deal with Clarity Values in Typescript.
+tags:
+  - tutorial
```

Comment:
Should there be a new sidebar category for "Guides"?

Reply:
yeah, that's a good question! We currently seem to have 3 types of content: guides, tutorials, and references. Guides are assumed the "default" type and are not labeled right now. tutorials are labeled (tag + tutorial category in the nav). references are in their own category. We probably should revisit the behavior at some point but I'd not worry about it as part of the docs week